spec/forms/candidate_interface/gcse_missing_form_spec.rb

```diff
@@ -22,6 +22,7 @@
       qualification = application_form.application_qualifications.create!(
         level: 'gcse',
         subject: 'maths',
+        enic_reason: 'not_needed',
```

Comment:
Have we considered setting "not_needed" as a default on the enum? https://api.rubyonrails.org/v7.1.3.4/classes/ActiveRecord/Enum.html